backup.config-example

```diff
@@ -43,3 +43,9 @@ GHE_NUM_SNAPSHOTS=10
 #
 # WARNING: do not enable this, only useful for debugging/development
 #GHE_BACKUP_FSCK=no
+
+# In some operatingsystems various tools such as gnu are located elsewhere
+# Modify you path here.
+#
+# Set this path on Solaris 11
+#PATH="/usr/gnu/bin:$PATH:/usr/sbin"
```

Comment:
I think it's best to make this as generic as possible without any references to specific operating systems.  I know Solaris puts the GNU utils here (I used to work for Sun and then Oracle on Solaris 😄) but we shouldn't mention this in the configuration example as it's not generic.

How about modifying this to:

```
# The PATH for the backup utilities to use. The default PATH used is that of the 
# executing user which may not always be customizable or contain all the utilities
# required by the backup utilities.
#
# PATH=""
```

This leaves things very generic and open to customisation that best suites the operating system or environment.